### PR TITLE
Update README and documentation

### PR DESCRIPTION
**Pull request by mfe**
Added : 
* requirements
* samples section
* 2017.1 section
* screenshots

For now, duplicated the work in the package doc. Future proof plan is to have a build script that copies the doc into the package.

See [README result](https://github.com/Unity-Technologies/usd-unity-sdk/blob/updateDoc/README.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,44 @@ This repository contains a set of libraries designed to support the use of
...
-easily import an export USD files.
+easily import and export USD files.
+
+![USD menu](Images/USD_menu.png)
+
+
+## Requirements
+
+* Windows / OSX
+* Unity version: 2018.3 (2018.3.4f1 and up) and 2019.1 (2019.1.0b2 and up)
```
Comment (clusty) [inline README.md]: I would only mention 2018.3.4f1 and up. Second part redundant ?

Comment (mfe) [reply README.md]: 2019.1 alphas and first beta will not work as the timeline package is either not here yet or with a version that not `1.0.0` yet.

Comment (clusty) [reply README.md]: Makes sense :)

**Comment by frenchdog** [comment]
Very happy to see how things are moving forward with the USD SDK! :smile:

**Comment by jcowles** [comment]
Getting close now!

Couple things I realized that should be added, one is linear color space:
image

For unlimited UsdSkel skinning weights in Unity 2019 (requires Unity 2019), this must be explicitly enabled (without this, rigs will be limited to 4-weights per vertex):
image

And here's one more screenshot of the .NET settings, just to make them all consistent:
image

**Comment by jcowles** [comment]
Oh, ha, I just noticed my screenshots are using the light editor skin.
Kind of superficial, but would someone mind re-capturing these with the dark skin?

**Comment by jcowles** [comment]
Actually, for the sake of expediency, I'm just going to merge this and update the images in a separate change -- but let me know if you have any opinions about the new images!

**Pull request merged**